source/manual/ab-testing.html.md

```diff
@@ -75,6 +75,19 @@ Follow these steps:
 7. Use the [govuk\_ab\_testing gem][govuk_ab_testing] to serve different versions to your users. It can be configured with the analytics dimension selected in step 2.
 8. To activate or deactivate the test, or to change the B percentage, update your test in the cdn-configs repo and deploy the change.
 
+### 2.3 Checklist for disabling A/B testing
+
+Follow these steps:
+
+1. Remove your test from the [ab_tests configuration file][configuration-file] in the [fastly-configure][fastly-configure] repo.
+2. Deploy the Fastly configuration to staging and production using the `Deploy_CDN` Jenkins job. The `vhost` must be set to `www`, and the credentials are in the deployment repo.
+3. Remove your test from the following files in [the cdn-configs repo][cdn-configs]:
```

Comment:
This part is maybe better in the readme of `cdn-configs`, so it doesn't get out of date. We can link it from here.